index.html

```diff
@@ -33,6 +33,105 @@ <h1 class="header2">I’m Iptisam
                 <li class="shape10"><a href="#"><img src="Images/Vector(1).jpg" alt="shape"></a></li>
                 <li class="shape11"><a href="#"><img src="Images/Vector(3).jpg" alt="shape"></a></li>
             </ul>
-        </section>  
+        </section>
+        <section class="container">
+            <div class="container1">
+                <div class="rect1"></div>
+                <h2 class="Proj1">Tonic</h2>
+                <ul class="list1">
+                    <li class="frame"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang">
+                    <li class="HTML"><a href="#">html</a></li>
+                    <li class="css"><a href="#">CSS</a></li>
+                    <li class="Java"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box">
+                <li class="button"><a href="#">See Project</a></li>
```

Comment:
- [ ] Kindly ensure all your buttons have button styling: active hover and pressed. below is what they should look like:
![image](https://user-images.githubusercontent.com/61408860/185750226-99398edd-1ec0-4673-adea-66c709e0898a.png)